vue/sbc-common-components/src/components/BaseAddress.vue

```diff
@@ -291,6 +261,15 @@ export default class BaseAddress extends Mixins(ValidationMixin, CountriesProvin
     return Boolean(this.schemaLocal && this.schemaLocal[prop] && this.schemaLocal[prop].isBC)
   }
 
+  /**
+   * Array of validation rules used by input elements to prevent extra whitespace.
+   */
+  private readonly spaceRules: Array<Function> = [
+    v => !/^\s/g.test(v) || 'Invalid spaces', // leading spaces
```

Comment:
I will try to handle leading and trailing spaces in one test statement ;just to avoid the two tests

Reply:
Yes, I could combine the 2 statements, but I think this is much easier to understand (and review) and for that reason I'd like to leave it as-is. I hope you're ok with that :)